src/coord/src/timestamp.rs

```diff
@@ -698,6 +698,7 @@ fn parse_debezium(record: Vec<(String, Value)>) -> Vec<(String, i64)> {
 /// Ex: last processed offset is 0 (ak, no records have been timestamped yet)
 /// The current max kafka offset is 0 (ak, the stream is empty). The function will return
 /// 0.
+#[allow(dead_code)]
```

Comment:
why can't we delete this if it is dead code?

Reply:
I wanted to delete it in a separate clean-up PR to keep this one simple.